addons/l10n_uy/demo/account_demo.py

```diff
@@ -495,13 +497,14 @@ def _l10n_uy_get_demo_data_res_partner(self, company=False):
                 'city': 'Montevideo',
                 'state_id': 'base.state_uy_10',
                 'country_id': 'base.uy',
+                'email': 'correo@example.com',
             },
-            # Foreign Partner
+            # Foreign Company
             'res_partner_foreign': {
                 'name': 'Foreign Inc',
-                'l10n_latam_identification_type_id': 'l10n_uy.it_nie',
+                'l10n_latam_identification_type_id': 'l10n_latam_base.it_vat',
                 'is_company': True,
-                'vat': '93:402.010-1',
+                'vat': 'AYXJA-1203812-053',
```

Comment:
@fvz-odoo "It is a 9-digit number, beginning with the number "9", formatted like an SSN (NNN-NN-NNNN)"  I have never had a good eye on how it looks like in the US and I wonder if it makes sense for the other demo data as well.

Reply:
Hi @jco-odoo and  @fvz-odoo 
I did put anything there, did not know the format of the US vat.
If you like to gitve a valid number I will put it in the data